CMakeLists.txt

```diff
@@ -121,6 +123,23 @@ endif()
 
 include(GNUInstallDirs)
 
+# ====================
+# === Dependencies ===
+# ====================
+
+include(FetchContent)
+FetchContent_Declare(
```

Comment:
I know it's "documenting the standard library", but we should have a comment about how to point this at a pre-cloned copy of fmt.